pallets/sgx-account-linker/src/weights.rs

```diff
@@ -35,64 +35,64 @@
 // --output=./pallets/account-linker/src/weights.rs
 // --template=./.maintain/frame-weight-template.hbs
 
-
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
-use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use frame_support::{
+    traits::Get,
+    weights::{constants::RocksDbWeight, Weight},
+};
 use sp_std::marker::PhantomData;
 
 /// Weight functions needed for pallet_account_linker.
 pub trait WeightInfo {
-	fn link_eth() -> Weight;
-	fn link_btc() -> Weight;
-	fn link_polkadot() -> Weight;
-	fn accept_polkadot() -> Weight;
+    fn link_eth() -> Weight;
+    fn link_btc() -> Weight;
+    fn link_polkadot() -> Weight;
+    fn accept_polkadot() -> Weight;
```

Comment:
There are still several places using the key word 'polkadot', please search and fix all of them.

Reply:
fixed